spring-ai-core/src/main/java/org/springframework/ai/chat/client/advisor/api/AfterAdvisor.java

```diff
@@ -33,23 +31,20 @@
  * @author Christian Tzolov
  * @since 1.0.0
  */
-public interface ResponseAdvisor extends Advisor {
+public interface AfterAdvisor extends Advisor {
 
 	/**
 	 * @param response the {@link ChatResponse} data to be advised. Represents the row
 	 * {@link ChatResponse} data after the {@link ChatModel#call(Prompt)} method is
 	 * called.
-	 * @param adviseContext the shared data between the advisors in the chain. It is
-	 * shared between all request and response advising points of all advisors in the
-	 * chain.
 	 * @return the advised {@link ChatResponse}.
 	 */
-	ChatResponse adviseResponse(ChatResponse response, Map<String, Object> adviseContext);
+	AdvisedResponse afterCall(AdvisedResponse advisedResponse);
```

Comment:
This name might be better aligned with `BeforeAdvisor`'s `before()` signature if the `Call` part is skipped, or the other one also has the `Call` suffix.